install.sh

```diff
@@ -281,7 +281,7 @@ verify_firewall_cmd() {
     # Check if exists
     if command -v "$_cmd" > /dev/null 2>&1; then
       # Found
-      FIREWALL=$_cmd
+      FIREWALL=$(command -v "$_cmd" 2>/dev/null)
```

Comment:
Remove.
I prefer `FIREWALL` to remain the simple name (as `DOWNLOADER`) and not the full path.